tests/X86/Run.cpp

```diff
@@ -458,6 +490,14 @@ static void RunWithFlags(const test::TestInfo *info,
   memset(lifted_state->x87.fxsave.st, 0, kill_size);
   memset(native_state->x87.fxsave.st, 0, kill_size);
 
+#if 32 == ADDRESS_SIZE_BITS
+  // If FCS and FDS are deprecated, don't compare them.
+  if (AreFCSAndFDSDeprecated()) {
```

Comment:
`if (32 == ADDRESS_SIZE_BITS && ...)`, that way we don't get a compiler warning about `AreFCSAndFDSDeprecated` being unused.